### PR TITLE
https://issues.redhat.com/browse/ACM-6348--known-issue-ingor…

### PR DESCRIPTION
**Pull request by swopebe**
…e-flag

### CODE REVIEW

File: release_notes/known_issues_governance.adoc [hunk 1]
```diff
@@ -27,6 +27,15 @@ For your {ocp} cluster, see link:https://access.redhat.com/documentation/en-us/o
...
 For more about deprecations and removals, see xref:../release_notes/deprecate_remove.adoc#deprecations-removals[Deprecations and removals].
 
+[#ingorePending]
+//2.8:6348
+
+== The _ignorePending_ flag is ignored by the policy generator
+
+The `ignorePending` flag is ignored if you set `consolidateManifests: true` in your policy generator.
+
+You can manually change the value if you need to implement the `ingorePending` function.
```
Comment (dhaiducek) [inline release_notes/known_issues_governance.adoc]:
```suggestion
You can set `consolidateManifests: false` if you need to implement the `ignorePending` function.
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks @swopebe! I have a minor suggestion/clarification.

**Comment by dhaiducek** [comment]
Also, I'm assuming that URL in the PR title was meant to be a Jira issue instead of a different PR?

**Review by dhaiducek — APPROVED** [review]
LGTM! Thanks, @swopebe!

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**